poplar.js

```diff
@@ -115,8 +118,22 @@ function elementify(html) {
   return div;
 }
 
-function getProp(object, path) {
-  return path && getDeep(object, path.split('.'));
+function getProp(item, path) {
+  if (!path) {
+    return;
+  }
+
+  var parts = path.split('.');
+
+  // Fast paths
+  if (parts.length == 1) {
+    return item[parts[0]];
+  }
+  if (parts.length == 2) {
+    return item[parts[0]][parts[1]];
+  }
```

Comment:
NIT: Single line `if` look cleaner to me. Might just be personal preference though :)

```js
if (!path) return;
var parts = path.split('.');

// Fast paths
if (parts.length == 1) return item[parts[0]];
if (parts.length == 2) return item[parts[0]][parts[1]];

return getDeep(item, parts);
```